volttron/platform/__init__.py

```diff
@@ -183,7 +183,7 @@ def is_instance_running(volttron_home=None):
     if volttron_home is None:
         volttron_home = get_home()
 
-    instance_file = os.path.expanduser("~/.volttron_instances")
+    instance_file = os.path.join(volttron_home, ".volttron_instances")
```

Comment:
This isn't if volttron_home exists.  This should be looking at the home directory .volttron instances file.  If it exist then we look at it and make sure that the current volttron home exists. and had a running system.